client/my-sites/upgrades/components/domain-warnings/style.scss

```diff
@@ -0,0 +1,12 @@
+.domain-warnings__unverified-domains {
+	a {
```

Comment:
IMHO this style is unnecessary. We already have a style for inline links in notices and they should be bold to be a bit more distinctive. And the `text-decoration` rule causes a double underlining in Chrome:
<img width="215" alt="screen shot 2016-03-26 at 19 26 02" src="https://cloud.githubusercontent.com/assets/3392497/14061764/e48a957e-f388-11e5-9d11-0489c0295be1.png">

There's an open issue about the [underline missing in Safari](https://github.com/Automattic/wp-calypso/issues/4321). Either way, you should rely on the global styles for the proper styling of `<a>` in notices.